JobConfig/primary/flateminus.fcl

```diff
@@ -1,8 +1,21 @@
-# flat range of eminus near the DIO endpoint
-#include "Production/JobConfig/primary/flatE.fcl"
-process_name: flateminus
-physics.producers.generate.physics.genId : dioTail
-physics.producers.generate.physics.pdgId : 11
-physics.producers.generate.physics.elow : 75.0 // Mev
-physics.producers.generate.physics.ehi : 110.0 // Mev
+#
+# generate and produce Detector Steps from flat spectrum using mu- target stops
+#
+# original author: S Middleton
+#
+#include "Production/JobConfig/primary/TargetStopParticle.fcl"
+physics.producers.generate : {
+  module_type : FlatMuonDaughterGenerator
+  inputSimParticles: TargetStopResampler
+  stoppingTargetMaterial : "Al"
+  verbosity : 0
+  makeHistograms : true
+  processcode : "mu2eMuonDecayAtRest"
+  pdgId : -11
+}
+
+physics.producers.generate.startMom : 0
+physics.producers.generate.endMom : 100
```

Comment:
This range doesn't cover all the physics use cases, in particular it's not sufficient to study DIO tail.  I wonder if these should be set to @nil, and require downstream users to pick a range based on their physics need.

Reply:
I like that idea.